data/redisrepo/remove_order.go

```diff
@@ -36,7 +36,9 @@ func (r *redisRepository) RemoveOrder(ctx context.Context, order models.Order) e
 
 	// update order status to CANCELED
 	orderIDKey := CreateOrderIDKey(order.Id)
-	transaction.HSet(ctx, orderIDKey, "status", models.STATUS_CANCELLED.String())
+	//transaction.HSet(ctx, orderIDKey, "status", models.STATUS_CANCELLED.String())
```

Comment:
Can you remove the comments here if we are now removing orders entirely when cancelled?